cli/mysql.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/cheggaaa/pb"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/docker/docker/pkg/term"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+)
+
+func init() {
+	register("mysql", runMysql, `
+usage: flynn mysql console [--] [<argument>...]
+       flynn mysql dump [-q] [-f <file>]
+       flynn mysql restore [-q] [-f <file>]
+
+Options:
+	-f, --file=<file>  name of dump file
+	-q, --quiet        don't print progress
+
+Commands:
+	console  Open a console to a Flynn mysql database. Any valid arguments to mysql may be provided.
```

Comment:
May I throw this one in? MariaDB and MySQL are (in small details) not compatible to each other. With having MySQL as a name and using MariaDB in the background, you would block the route to having both integrated. I think it would make sense at some point, to offer both.